openstack/nova/templates/seed.yaml

```diff
@@ -21,6 +21,7 @@ spec:
   - monsoon3/domain-neo-seed
   - monsoon3/domain-s4-seed
   - monsoon3/domain-wbs-seed
+  - monsoon3/glance-seed
```

Comment:
If this is just for the role `cloud_image_admin`, then I think we should just add it down below to the `roles` instead of creating this dependency. We opted to do that for all other roles, too.